lib/BackgroundJob/Tasks/ImageProcessingTask.php

```diff
@@ -78,10 +78,16 @@ public function do(FaceRecognitionContext $context) {
 			$userRoot = $userFolder->getParent();
 			$file = $userRoot->getById($image->file);
 			// todo: handle case where file do not exists anymore
+			// todo: this concat is wrong with shared files.
 			$imagePath = $dataDir . $file[0]->getPath();
 			$this->logInfo('Processing image ' . $imagePath);
 			list($tempfilePath, $ratio) = $this->prepareImage($imagePath);
 
+			if ($tempfilePath == null || $ratio <= 0.0) {
+				$this->logInfo('There was some error with the file, continue with the next one');
```

Comment:
in some future version, we should update Image to add it is a) processed, b) stack trace to it (so it doesn't always go over same file). But, yes, it is OK to unblock both you and me now!